examples/nextjs-app/app/todos/[id]/page.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+
+import { Todo, todos } from '@/app/mockData/todos';
+import {
+  DatePicker,
+  DynamicPage,
+  DynamicPageTitle,
+  Form,
+  FormItem,
+  Input,
+  MessageStrip,
+  MessageStripDesign,
+  Switch,
+  TextArea
+} from '@ui5/webcomponents-react';
+
+export default async function TodoDetails({ params }: { params: { id: string } }) {
+  // this is a very simple mock which mimics data fetching
+  const todo = await new Promise<Todo | undefined>((resolve) => {
+    setTimeout(() => {
+      resolve(todos.at(Number(params.id)));
+    }, 500);
+  });
+
+  return (
+    <>
+      <DynamicPage showHideHeaderButton={false} headerTitle={<DynamicPageTitle header={todo?.title} />}>
+        <MessageStrip design={MessageStripDesign.Information}>
+          {`Since this is only a demo app, adjustments made here on this page won't be reflected in the todo list.`}
```

Comment:
Why use a template string here?

Reply:
the next eslint config complains about the unescaped `'` in this line, so we would have to go either for `&apos;` or use template string to avoid this error. Using a template string sounded more convenient to me.

Reply:
You're right, I think eslint for React (or js in general) does the same.